feature-libs/organization/administration/components/unit/details/unit-details.component.html

```diff
@@ -1,12 +1,19 @@
 <cx-org-card *ngIf="model$ | async as model" i18nRoot="orgUnit.details">
-  <a
+  <div
```

Comment:
I'd not introduce a new DOM element here, you can apply the condition and config directly to the anchor.

Reply:
Hi Tobias, 

Thanks a lot for your kind review! Let me explain why I introduce a wrapper div element on purpose.

Yes, I also observed that the host element which declares [cxFocus] directive will have tabindex set as -1. 

I debugged and knew the reason, tabindex is set as -1 by default in base-focus.directive.ts:

![image](https://user-images.githubusercontent.com/58975336/103994974-bb29e880-51d2-11eb-8330-0c21290cd5cb.png)

In order to avoid this issue, I introduce a wrapper div element, and declare [cxFocus] directive on it. In this way, div itself has tabindex = -1, while edit button, the a tag, is still focusable:

![image](https://user-images.githubusercontent.com/58975336/103995256-3095b900-51d3-11eb-9b14-6ffb86496df6.png)

So do you know if there's any approach, to avoid the "side-effect" of having host element's tabindex attribute set as default value -1? I do believe there's a way, otherwise it does not make sense. 

![lock+Focus (1)](https://user-images.githubusercontent.com/58975336/103995488-85393400-51d3-11eb-9edd-038d29bf1fcc.png)

I tried to find some hint in lock-focus.directive.ts but unfortunately didn't find what I want - in my eye its logic is complex, I draw a mindmap to try to understand it...

Thanks a lot for your help!

Reply:
understood. I'd still like to avoid the div, can you debut on why `requiresExplicitTabIndex` is returning `true` in this case?

Reply:
Hi Tobias, 

I am fine with removal of this unnecessary div wrapper. Now <a> tag looks like below. I didn't add refresh function since it is not related to "tabindex = -1" trouble shooting.

```html
<a
      actions *ngIf="model.active"
      class="link edit"
      [class.disabled]="!model.active || (isInEditMode$ | async)"
      [routerLink]="{ cxRoute: 'orgUnitEdit', params: model } | cxUrl"
      [cxFocus]="{ autofocus: true }"
    >
      {{ 'organization.edit' | cxTranslate }}
    </a>
```

when debugging into function **requiresExplicitTabIndex**, it returns true because evaluation '**(this.host.hasAttribute('href') || this.host.hasAttribute('routerlink'))**' returns false.

![image](https://user-images.githubusercontent.com/58975336/104019239-289b4080-51f6-11eb-88e1-7bdd11aff871.png)

In the context of **requiresExplicitTabIndex** execution, <a> tag does not have href attribute populated yet.

Reply:
Good summary, and i'm not against using that angular internal to be frank. We could do a better job, but it takes a refactor that we cannot easily do without breaking changes. I've used the ng internal for now to unblock us with this behaviour. It's hitting us in more locations, so i'm eager to get this fixed.